src/sdk/pynni/nni/compression/torch/compressor.py

```diff
@@ -149,13 +149,18 @@ def select_config(self, layer):
         ret = None
         for config in self.config_list:
             config = config.copy()
-            config['op_types'] = self._expand_config_op_types(config)
-            if layer.type not in config['op_types']:
+            # expand config if key `default` is in config['op_types']
+            if 'op_types' in config and 'default' in config['op_types']:
+                config['op_types'].extend(default_layers.weighted_modules)
```

Comment:
better to remove 'default' after it is expanded.

Reply:
I think it is best to just drop 'default' in op_types.

Reply:
fixed.